keras_cv/losses/iou_loss_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.losses.iou_loss import IoULoss
+
+
+class IoUTest(tf.test.TestCase):
+    def test_output_shape(self):
```

Comment:
can you add test cases over the reduction strategies, in particular `reduction='none'` is good to test.

Reply:
`test_output_shape_reduction_none` covers this, right?